itou/users/enums.py

```diff
@@ -50,16 +50,14 @@ class IdentityProvider(models.TextChoices):
     PRO_CONNECT = "PC", "ProConnect"
     PE_CONNECT = "PEC", "Pôle emploi Connect"
 
-    @classmethod
-    @property
-    def supported_user_kinds(cls):
```

Comment:
Ce n'est plus autorisé avec python 3.13 ?

Reply:
Oui, comme noté dans le message de commit. https://docs.python.org/3/library/functions.html#classmethod

Deprecated since version 3.11, removed in version 3.13: Class methods can no longer wrap other descriptors such as property().

Reply:
Oups, désolé, j'ai pas pensé à aller voir le message de commit

Reply:
Après  on pourrait juste rajouter:
```
class classproperty:
    def __init__(self, func):
        self.fget = func
    def __get__(self, instance, owner):
        return self.fget(owner)
```
pour ne pas avoir à changer le reste et continuer à avoir des "property" au niveau de la classe...

Reply:
Bof, les properties sur une enum ne me semblent pas trop être l’idée d’une enum. Même si je reconnais que c’est pratique.